client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -601,3 +601,71 @@ def test_update_twice(self, client, registered_model, created_endpoints):
         endpoint.update(new_model_version, DirectUpdateStrategy(), wait=True)
         test_data = np.random.random((4, 12))
         assert np.array_equal(endpoint.get_deployed_model().predict(test_data), new_classifier.predict(test_data))
+
+    def test_update_from_run_diff_workspace(self, client, organization, created_endpoints, experiment_run, model_for_deployment):
```

Comment:
I'm going to assume that `client` wasn't used to create the other objects in this argument. Let me know if I'm wrong.

Reply:
Actually both are created by the same client, but has different workspaces: one in a personal workspace, the other in an organizational workspace.

Reply:
There's a test for two objects created by two different clients below.